x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/OperatorPrivilegesIT.java

```diff
@@ -100,4 +99,86 @@ public void testOperatorPrivilegesXpackUsage() throws IOException {
         assertTrue((boolean) operatorPrivileges.get("available"));
         assertTrue((boolean) operatorPrivileges.get("enabled"));
     }
+
```

Comment:
I think we should just remove this and the associated build file changes etc. from this PR? Once  the real framework for avoiding to restore autoscaling policies on operator privilege enabled setups goes in, this might look different anyway.

Reply:
Agreed and all unrelated changes removed.

A bit more context: I thought about deleting these changes in last commit. But I got a bit paranoid and was trying to assert that operator privileges do not interfere with the restore behaviour. Looking back now, this thinking was an artefact of how this PR evolved, which started with the assumption that operator privileges would impact the behaviour.  If it had started purely from the autoscaling side, I don't think I would add anything related to operator privileges. In short, it's the right move to delete these unrelated code.